### ISSUE TITLE
Meet 1.4.6 Contrast (Enhanced)

### ISSUE THREAD

**Issue by joshdanielson** [loconomics/loconomics#619]
## Summary
The visual presentation of text and images of text has a contrast ratio of at least 7:1, except for the following:   
- Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 4.5:1;
- Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
- Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

**Conformance Level:** AAA (See #518 for level AA)
**Existing 508 Corresponding Provision:** None

### ### Small text (under 18 point regular font or 14 point bold font) SHOULD have a contrast ratio of at least 7 to 1 with the background.

Consider increasing the color contrast beyond the minimum requirements for mobile applications. Due to the smaller viewport size and varying environmental conditions that devices are used in, enhanced contrast will make an app the most accessible to the widest number of users. A contrast ratio of 7:1 can compensate the contrast loss for people whose vision is about 20/80 or those with color vision deficiencies. These users typically don't need to use contrast-enhancing assistive technologies, but may have difficulty seeing contrast that is 4.5:1, especially on mobile devices.

**Comment by joshdanielson** [comment]
Met as part of #528. Closing.
